test/layer2libTest.js

```diff
@@ -28,15 +28,15 @@ async function test(redisClient) {
   // ALICE
   let optionsAlice = {
     db: redisProxy,
-    privateKey: '0x6cb2b4257e4477b096beacc755b6abf45d9d67738522aa27d3c2e1444eb4ea80'
```

Comment:
we should probably leave a test account in the readme or use the same private keys so everyone can test from the same keys until there is a better testing setup for injecting the private key into layer2lib